wasi/example_test.go

```diff
@@ -1,14 +1,49 @@
 package wasi
 
-import "github.com/tetratelabs/wazero"
+import (
+	_ "embed"
+	"fmt"
+	"log"
+	"os"
 
-var r = wazero.NewRuntime()
+	"github.com/tetratelabs/wazero"
+	"github.com/tetratelabs/wazero/sys"
+)
 
-// Example_InstantiateSnapshotPreview1 shows how to instantiate ModuleSnapshotPreview1, which allows other modules to
-// import functions such as "wasi_snapshot_preview1" "fd_write".
-func Example_instantiateSnapshotPreview1() {
-	wm, _ := InstantiateSnapshotPreview1(r)
+// This is an example of how to use WebAssembly System Interface (WASI) with its simplest function: "proc_exit".
```

Comment:
This also has overlap with the examples directory, but we should have it to seed one example into https://pkg.go.dev/github.com/tetratelabs/wazero/wasi